composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "neos/neos": ">3.3"
+        "neos/neos": ">=4.3"
```

Comment:
I'd rather use `^4.3 | ^5.0` to prevent incompatibilities like the ones caused by `>` (which will haunt this package forever, but will obviously need less maintenance if it will be compatible with f.e. Neos 6.0).